CONTRIBUTING.md

```diff
@@ -2,71 +2,20 @@
 
-### Additional information
-
-In **[our Wiki](https://github.com/CMSgov/design-system/wiki)** you can find additional information like:
+Use **[our Wiki](https://github.com/CMSgov/design-system/wiki)** to find additional information like:
 
 * Pattern proposal process
 * Coding guidelines
 * Guiding principles
 * How to write documentation
 * etc.
 
-## Running locally
-
-This project uses [Yarn](https://yarnpkg.com/) for package management. Yarn helps ensure everyone is using the same package versions. If you've used NPM before, you'll have no trouble using Yarn.
-
-[**Install Yarn**](https://yarnpkg.com/docs/install), if you don't have it yet.
-
-### Getting started
-
-1. `yarn install`
-   * This will also run [Lerna](https://lernajs.io/) `bootstrap` which allows us to have multiple packages within the same repo (a monorepo). Lerna installs all our dependencies and symlinks any cross-dependencies.
-1. `yarn start`
-
-_Note_: When you create a Git commit, any staged scripts will be automatically ran through ESLint and Prettier. If the linter catches an error, your commit will fail. This is a feature, not a bug :)
-
-### Scripts
-
-These scripts can all be run from the root level of the repo:
-
-* `yarn start`
-  * Starts local server running the documentation site
-  * Regenerates documentation when files change
-* `yarn build`
-  * Compile/transpile/uglify everything and makes things release-ready.
-* `yarn bump`
-  * Increments package versions. Read "[Versioning](https://github.com/CMSgov/design-system/wiki/Versioning)" for more info.
-* `yarn generate`
-  * Generates the necessary files for a new core component
-  * Alias: `yarn g`
-* `yarn test`
-  * Runs JS unit tests
-  * Lints JS using ESLint
-  * Lints Sass using stylelint
-* `yarn test:watch`
-  * Runs JS unit tests and will continue to run tests as files change
-* `yarn update-snapshots`
-  * Updates [Jest snapshots](http://facebook.github.io/jest/docs/en/snapshot-testing.html)
-* `yarn lint`
-  * Runs just the linting portion of the tests
-
-#### Theme scripts
-
-You can also use the following scripts to [preview and build a theme](https://github.com/CMSgov/design-system/wiki/site-packages-and-themes):
-
-* `yarn start:theme`
-* `yarn build:theme`
-
-These scripts will also place the documentation into a `docs` subdirectory in your theme's directory.
-
-If you have multiple directories inside of `packages/themes`, you can specify which theme to use by passing the scripts the name of the folder. For example: `yarn start:theme my-theme-folder-name`
-
-If your documentation site will be uploaded to a subdirectory (ie. example.com/design-system), you can set its root path by passing the `--root` option. For example: `yarn build:theme --root design-system`
+Below you'll find information onhow to contribute code to the design system.
 
 ## Submitting a pull request
 
```

Comment:
What do you think about adding a step to the PR instructions?

* Create a feature branch from master: `git checkout -b username/branch-name` in your terminal
* Commit your changes locally: `git commit -a -m "YOUR MESSAGE"`
* Push your changes to the remote repo: `git push origin username/branch-name`
* Make a [pull request](https://help.github.com/en/articles/creating-a-pull-request) against the master branch

I feel we'd be doing users just getting started with Git a service by explaining each command and offering full command syntax.

Reply:
I like this idea, what about linking out to GitHub documentation instead?
https://help.github.com/en/articles/creating-a-pull-request#creating-the-pull-request

Reply:
Also do we need this sentence? I think we could remove it. 
> Below you'll find information on how to contribute code to the design system.

Reply:
@line47 I like the idea to link to the GH documentation on how to make a pull request. Let's do that and merge this thing!